sci-biology/spm/ChangeLog

```diff
@@ -0,0 +1,7 @@
+# ChangeLog for sci-biology/spm
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  17 Feb 2015;  <chymera@gentoo.org> +metadata.xml, +spm-8.ebuild.orig,
```

Comment:
and then what? won't that set the variable just for my session? I this is an other repoman issue? where does repoman take my name and email from?

Perhaps we could also just let this rest? I don't mind if I am not credited for this particular thing. I feel we're getting too sidetracked by metadata....